pkgs/development/python-modules/m2crypto/default.nix

```diff
@@ -16,6 +17,14 @@ buildPythonPackage rec {
     sha256 = "a1b2751cdadc6afac3df8a5799676b7b7c67a6ad144bb62d38563062e7cd3fc6";
   };
 
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/void-linux/void-packages/raw/7946d12eb3d815e5ecd4578f1a6133d948694370/srcpkgs/python-M2Crypto/patches/libressl.patch";
```

Comment:
Is there a patch in the `m2crypto` repo?

Reply:
@FRidh - No; the latest version [on `master`](https://github.com/mcepl/M2Crypto/blob/master/SWIG/_bio.i#L296-L299) does not contain anything referencing libressl.

Reply:
there is an opposing stance https://gitlab.com/m2crypto/m2crypto/merge_requests/139

Reply:
@andrew-d please add yourself as maintainer then. Patch itself looks OK to me.

Reply:
@danbst - Done!